source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -228,13 +229,31 @@ def _get_columnCount(self):
 			return 1
 		return count
 
-	def _get__columnOrderArray(self):
-		coa=(c_int *self.columnCount)()
+	def _getColumnOrderArrayRaw(self, columnCount: int):

```

Comment:
can this (and other getColumnOrderArray functions) get a return type added?

Reply:
I'm not sure how since we're using a dynamic python class (ctypes array) as the return type.

Reply:
I think `ctypes.Array` works, however I could stand corrected.
Note that the following evaluates to `True`

```
import ctypes
columnOrderArray = (ctypes.c_int * 7)()
isinstance(columnOrderArray, ctypes.Array)
```